<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>172</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mount a LUKS device</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/icon.png</normaloff>../icons/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mount Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnminimumwidth="0,0,0">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Device:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineDevice"/>
      </item>
      <item row="0" column="2">
       <widget class="QDialogButtonBox" name="selectDevice">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Open</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mountpoint:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineMount"/>
      </item>
      <item row="1" column="2">
       <widget class="QDialogButtonBox" name="selectMount">
        <property name="standardButtons">
         <set>QDialogButtonBox::Open</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Filesystem:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ext4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ext3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ext2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>jfs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>xfs</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
